actions/workflows/bwc_prep_dev.yaml

```diff
@@ -1,124 +1,153 @@
+      hosts: <% ctx().host %>
+      cwd: <% ctx().cwd %>
+    next:
+      - when: <% succeeded() %>
+        do:
+          - prep_st2enterprise_ldap
+  prep_st2enterprise_ldap:
+    action: st2cd.st2_prep_dev_for_enterprise_ldap
+    input:
+      project: st2-enterprise-auth-backend-ldap
+      version: <% ctx().dev_version %>
+      fork: <% ctx().fork %>
+      local_repo: <% 'st2enterprise_auth_ldap_' + ctx().local_repo_sfx %>
+      hosts: <% ctx().host %>
+      cwd: <% ctx().cwd %>
+    next:
+      - when: <% succeeded() %>
+        do:
+          - prep_bwc_ui
+  prep_bwc_ui:
+    action: st2cd.bwc_prep_dev_for_bwc_ui
+    input:
+      project: bwc-ui
+      version: <% ctx().dev_version %>
+      fork: <% ctx().fork %>
+      local_repo: <% 'bwc_ui_' + ctx().local_repo_sfx %>
+      hosts: <% ctx().host %>
+      cwd: <% ctx().cwd %>
+    next:
+      # Commented out in the original workflow
+      # - when: <% succeeded() %>
+      #   do:
+      #     - prep_bwc_ipfabric_suite
+      - when: <% failed() %>
+        do:
+          - cleanup_on_failure
+  # Commented out in the original workflow
+  # prep_bwc_ipfabric_suite:
+  #   action: st2cd.bwc_prep_dev_for_bwc_ipfabric_suite
+  #   input:
+  #     version: <% ctx().version %>
+  #     fork: <% ctx().fork %>
+  #     local_repo: <% 'bwc_ipfabric_suite_' + ctx().local_repo_sfx %>
+  #     hosts: <% ctx().host %>
+  #     cwd: <% ctx().cwd %>
+  #   next:
+  #     - when: <% succeeded() %>
+  #       do:
+  #         - prep_bwc_ipfabric_packs
+  #     - when: <% failed() %>
+  #       do:
+  #         - cleanup_on_failure
+  # prep_bwc_ipfabric_packs:
+  #   action: st2cd.bwc_prep_dev_for_bwc_ipfabric_packs
+  #   input:
+  #     version: <% ctx().dev_version %>
+  #     fork: <% ctx().fork %>
+  #     local_repo: <% 'bwc_ipfabric_packs_' + ctx().local_repo_sfx %>
+  #     hosts: <% ctx().host %>
+  #     cwd: <% ctx().cwd %>
+  #   next:
+  #     - when: <% succeeded() %>
+  #       do:
+  #         - prep_bwc_topology
+  #     - when: <% failed() %>
+  #       do:
+  #         - cleanup_on_failure
+  # prep_bwc_topology:
+  #   action: st2cd.bwc_prep_dev_for_bwc_topology
+  #   input:
+  #     version: <% ctx().dev_version %>
+  #     fork: <% ctx().fork %>
+  #     local_repo: <% 'bwc_topology_' + ctx().local_repo_sfx %>
+  #     hosts: <% ctx().host %>
+  #     cwd: <% ctx().cwd %>
+  #   next:
+  #     - when: <% succeeded() %>
+  #       do:
+  #         - prep_bwc_cli
+  #     - when: <% failed() %>
+  #       do:
+  #         - cleanup_on_failure
+  # prep_bwc_cli:
+  #   action: st2cd.bwc_prep_dev_for_bwc_cli
+  #   input:
+  #     version: <% ctx().dev_version %>
+  #     fork: <% ctx().fork %>
+  #     local_repo: <% 'bwc_cli_' + ctx().local_repo_sfx %>
+  #     hosts: <% ctx().host %>
+  #     cwd: <% ctx().cwd %>
+  #   next:
+  #     - when: <% failed() %>
+  #       do:
+  #         - cleanup_on_failure
```

Comment:
Let's just remove commented out code since we're touching this.

Reply:
Fixed in 8bfeff9.